solutions/cses-1076.mdx

```diff
@@ -87,3 +82,40 @@ int main() {
 	cout << endl;
 }
 ```
+
+## Solution 2: Order Statistic Tree
+(Arpan Banerjee)
```

Comment:
idt adding your name here is necessary; it's already in the authors list

Reply:
ye not necessary